### ISSUE TITLE
redwood install fails

### ISSUE THREAD

**Issue by briandoconnor** [BD2KGenomics/dcc-ops#62]
Hey Beni, Carlos and I are testing dcc-ops and I’m running into a problem with redwood, it’s missing redwood-install.  This line of install_bootstrap is failing: `sudo admin/bin/redwood install`

On the feature branch: https://github.com/BD2KGenomics/dcc-ops/tree/feature/brian_carlos_a2_testing

I commented out this line so we could move forward

**Comment by benjaminran** [comment]
This is my mistake - I removed redwood install in favor of a docker restart
policy (thanks Carlos) but forgot to remove it from the installer. The
redwood install call should be deleted from install_bootstrap.

On May 6, 2017 1:41 PM, "briandoconnor" <notifications@github.com> wrote:

Hey Beni, Carlos and I are testing dcc-ops and I’m running into a problem
with redwood, it’s missing redwood-install. This line of install_bootstrap
is failing: sudo admin/bin/redwood install

On the feature branch: https://github.com/BD2KGenomics/dcc-ops/tree/
feature/brian_carlos_a2_testing

I commented out this line so we could move forward

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/BD2KGenomics/dcc-ops/issues/62>, or mute the thread
